### ISSUE TITLE
телепорт роев через шов мира

### ISSUE THREAD

**Issue by allsteak** [KranX/Vangers#235]
https://www.youtube.com/watch?v=V75ywaJnFyQ

на видео показано, как в HD версии рой прыгает с одной половины экрана на другую при пересечении отметки в полпути между этими (этой?) комиями роя.

тобишь в случае, когда одна и та же координата попадает на экран дважды - отображается только брижайший к нам рой.

**Comment by klimrmad** [comment]
Это не телепорт а один и тот же объект. По идее точно так же будут прыгать и другие объекты. Хороший повод ограничить ширину экрана до ширины мира.

**Comment by stalkerg** [comment]
Я согласен, с точки зрения игры это кажется слишком много свободы у HD версии. Каждый объект должен рисоваться только один раз на экране.
